cylc/flow/task_pool.py

```diff
@@ -81,9 +88,32 @@
     from cylc.flow.workflow_db_mgr import WorkflowDatabaseManager
     from cylc.flow.flow_mgr import FlowMgr, FlowNums
 
+
 Pool = Dict['PointBase', Dict[str, TaskProxy]]
 
 
+def prereqs_str_to_tokens(prereqs: List[str]) -> List[Tuple[str, str, str]]:
```

Comment:
Is this needed? Would it be easier to pass the Tokens object straight through to `satisfy_me`?

```suggestion
def prereqs_str_to_tokens(prereqs: Iterable[str]) -> List[Tuple[str, str, str]]:
```